### ISSUE TITLE
Add an option for disable auto hide controls

### ISSUE THREAD

**Issue by xmha97** [mpv-android/mpv-android#620]
Please add an option to disable auto hide controls.

![IMG_20221029_222859_404](https://user-images.githubusercontent.com/23485114/198848400-089b5362-f7ce-470a-9249-ec85896058b4.jpg)

**Comment by rohitghali** [comment]
Yep. Much needed. Everytime I use --sub-seek, the ui shows up. Bad UX.

**Comment by Sergei-Alekseev** [comment]
Yes, pls add it. Only one thing bother me in mpv - autohide of controls) i just hate it.
ui.controls_fade=0 in mpv.conf is simple solution i guess.

**Comment by chrcoluk** [comment]
where is mpv.conf stored on the android app?

**Comment by rohitghali** [comment]
> where is mpv.conf stored on the android app?

Just open mpv settings and choose advanced.
